src/d3a/models/area/redis_external_market_connection.py

```diff
@@ -155,20 +159,20 @@ def event_market_cycle(self):
         if self.area.current_market is None:
             return
         market_event_channel = f"{self.channel_prefix}/market-events/market"
-        current_market_info = self.area.next_market.info
-        current_market_info["current_market_fee"] = \
+        market_info = self.market.info
```

Comment:
```suggestion
        market_info = self.area.next_market.info
```
I guess

Reply:
@faizan2590 now, please have a closer look, i created a `market` property, the same as in the external strategies.

Reply:
Ah yes, then ignore my comment but please don't be implicit in variable naming specially for markets, I would rather prefer to be explicit and name it `next_market `

Reply:
I used the naming from the strategies and found it good, as we will change this in in the frame of https://gridsingularity.atlassian.net/browse/D3ASIM-2955